### PR TITLE
Run3-gex138D Use getByToken instead of getByLabel in CaloOnlineTools/EcalTools

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Use getByToken instead of getByLabel in CaloOnlineTools/EcalTools

#### PR validation:

Use the runTheMatrix test workflows

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: CaloOnlineTools/EcalTools/plugins/EcalPnGraphs.h [hunk 1]
```diff
@@ -36,7 +36,7 @@ class EcalPnGraphs : public edm::one::EDAnalyzer<> {
 protected:
   //  std::string ebDigiCollection_;
```
Comment (ChrisMisan) [inline CaloOnlineTools/EcalTools/plugins/EcalPnGraphs.h:37]: Is this needed?

**Review by ChrisMisan — COMMENTED** [review]
- Commented code in CaloOnlineTools/EcalTools/plugins/EcalExclusiveTrigFilter.cc and CaloOnlineTools/EcalTools/plugins/EcalPulseShapeGrapher.cc should be removed.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by ChrisMisan** [comment]
+alca

- differences in MTD and L1T are not related to this PR

**Comment by perrotta** [comment]
+1

**Pull request merged**